wordle-web/components/keyboard.vue

```diff
@@ -13,28 +15,53 @@
         </v-container>
       </v-col>
     </v-row>
-    <v-btn
-      :disabled="wordleGame.gameOver"
-      class="float-left"
-      @click="guessWord"
-    >
-      Guess
-    </v-btn>
-    <v-btn
-      :disabled="wordleGame.gameOver"
-      icon
-      class="float-right"
-      @click="removeLetter"
-    >
-      <v-icon>mdi-backspace</v-icon>
-    </v-btn>
+    <v-layout align-space-between justify-space-between> 
+      <v-btn
+        :disabled="wordleGame.gameOver"
+        class="float-left"
+        @click="guessWord"
+      >
+        Guess
+      </v-btn>
+      <!--make scroll-->
+      <v-dialog
+        v-model="dialog"
+        scrollable
+        max-width="300px"
+      >
+        <template v-slot:activator="{ on, attrs }">
+          <v-btn
+            color="primary"
+            dark
+            v-bind="attrs"
+            v-on="on"
+          >
+          {{availableWords}} Available Words
+          </v-btn>
```

Comment:
the vuetify `v-dialog` doesn't have these `activator` slots. You'd have to make this button outside of the dialog, and upon clicking it, using `@click=` call a function to pop up the dialog.

It's unclear where you are triggering the calculation of the matching words, the variable `availableWords` doesn't seem to exist in this file?